### PR TITLE
chore(): use Isomer dependency instead of internal animation

### PR DESCRIPTION
**Pull request by liamdebeasi**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Build (`npm run build`) was run locally and any changes were pushed
- [ ] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

-
-
-

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by liamdebeasi** [comment]
Isomer version will be bumped to 1.0.0 prior to merging

File: angular/src/index.ts [hunk 1]
```diff
@@ -43,7 +42,10 @@ export * from './types/ionic-lifecycle-hooks';
...
 // UTILS
-export { IonicSafeString, getPlatforms, isPlatform, createAnimation } from '@ionic/core';
+export { IonicSafeString, getPlatforms, isPlatform } from '@ionic/core';
 
 // CORE TYPES
-export { Animation, AnimationBuilder, AnimationCallbackOptions, AnimationDirection, AnimationFill, AnimationKeyFrames, AnimationLifecycle, Gesture, GestureConfig, GestureDetail, mdTransitionAnimation, iosTransitionAnimation, NavComponentWithProps } from '@ionic/core';
+export { AnimationBuilder, Gesture, GestureConfig, GestureDetail, mdTransitionAnimation, iosTransitionAnimation, NavComponentWithProps } from '@ionic/core';
+
+// ISOMER
+export * from '@isomer/angular';
```
Comment (liamdebeasi) [inline angular/src/index.ts:51]: This works, but is it a good idea to do?

Comment (adamdbradley) [reply angular/src/index.ts:51]: Probably use named exports instead so we don't accidentally add something to the public api

File: packages/react/src/components/index.ts [hunk 2]
```diff
@@ -33,8 +33,8 @@ export * from './IonRouterContext';
 export { isPlatform, getPlatforms, getConfig, ionRenderToString } from './utils';
 export * from './hrefprops';
 
-// Ionic Animations
-export { CreateAnimation } from './CreateAnimation';
+// Isomer
+export * from '@isomer/react';
```
Comment (adamdbradley) [inline packages/react/src/components/index.ts:37]: named exports

**Comment by liamdebeasi** [comment]
Closing for now.

**Pull request closed without merging**